tests/test_business.py

```diff
@@ -0,0 +1,118 @@
+"""Contains test for the Business class"""
+
+import unittest
+
+from app.api_v1.models import Business, Review
+
+class TestBusinessCase(unittest.TestCase):
+    """Test Business class functionality
+    Attributes:
+        business(obj): a business object to be used for \
+        subsequent tests
+    Methods:
+        test_create_business()
+        test_cannot_create_duplicate_businesses()
+        test_create_business()
+        test_edit_business()
+        test_delete_business()
+        test_find_business_by_id()
+        test_find_businesses_by_category()
+        test_find_businesses_by_location()
+        test_write_review()
+        test_get_all_reviews()
+    """
+    def setUp(self):
+        self.mock_business = Business("James Barber", "We sell bananas", \
+        "Kawangware", "fruit vendors")
+        self.success_user = User('ptah', 'pndungu54@gmail.com', 'pass123', 'pass123')
+        
+    def tearDown(self):
+        del self.mock_business
+
+    def test_create_business(self):
+        """Test whether a business can be created succesfully"""
+        response = self.mock_business.create_business()
+        self.assertTrue(response)
+        self.assertEqual(Business.get_all_businesses(), \
+        [{"Name":"James Barber", "Descrition":"We sell bananas", \
+        "Location":"Kawangware", "Category":"fruit vendors"}])
+    
+    def test_cannot_create_duplicate_businesses(self):
```

Comment:
What do you think about breaking down the test case to test a unique key like "business name should not be a duplicate" rather than testing the whole business info? This is bacause some businesses can be in the same location and category.